src/opustrainer/modifiers/retokenize.py

```diff
@@ -8,27 +8,8 @@
 from opustrainer import logger
 
 
-def print_cells(*rows, format:Callable[[Any],str]=repr,file=None) -> None:
```

Comment:
Tbh I'm in favour of keeping those, even if it's in a separate file. At some point in the future it might be useful.

Reply:
I kept a copy here: https://gist.github.com/jelmervdl/3b45af92aaf91b75c3040a7f2824b973

To be honest, it didn't work all that well with unicode and my terminal. My terminal apparently renders emoji 2 characters wide. To deal with this stuff correctly, you need something like [wcwidth](https://man7.org/linux/man-pages/man3/wcwidth.3.html) ([for Python](https://pypi.org/project/wcwidth/))